### PR TITLE
🎉 New Source: Pendo [Configuration Based Source]

### PR DESCRIPTION
**Pull request by YiyangLi**
## What
- add a new source pendo, generated by the low-code platform

## How
- add 4 full import streams to bootstrap the new source connector
- add documentation
- add svg
- generate uuid and add a record in source_definitions.yaml
- issue: https://github.com/airbytehq/airbyte/issues/2977

## Recommended reading order
1. airbyte-integrations/connectors/source-pendo/source_pendo/manifest.yaml
2. airbyte-config/init/src/main/resources/icons/pendo.svg

## 🚨 User Impact 🚨
no, it's a new source connector

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [x] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [x] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details><summary><strong>Connector Generator</strong></summary>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed

</details>

### CODE REVIEW

**Comment by YiyangLi** [comment]
```
> python -m pytest integration_tests -p integration_tests.acceptance
Test session starts (platform: darwin, Python 3.9.13, pytest 6.2.5, pytest-sugar 0.9.6)
cachedir: .pytest_cache
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/.hypothesis/examples')
rootdir: /Users/bartdev/Playground/airbyte, configfile: pytest.ini
plugins: hypothesis-6.54.6, requests-mock-1.9.3, mock-3.6.1, sugar-0.9.6, timeout-1.4.2, cov-3.0.0
collecting ... {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_config_match_spec[inputs0] ✓                                                                                                                     3% ▍         {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_match_expected[inputs0] ✓                                                                                                                        6% ▋         {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_docker_env[inputs0] ✓                                                                                                                            9% ▉         {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_enum_usage[inputs0] ✓                                                                                                                           12% █▎        {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_oneof_usage[inputs0] ✓                                                                                                                          15% █▌        
 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_required[inputs0] ✓                                                                                                                             18% █▊        
 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_optional[inputs0] ✓                                                                                                                             21% ██▏       
 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_has_secret[inputs0] ✓                                                                                                                           24% ██▍       
 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_secret_never_in_the_output[inputs0] ✓                                                                                                           26% ██▋       {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_secret_is_properly_marked[inputs0] ✓                                                                                                            29% ██▉       
 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_property_type_is_not_array[inputs0] ✓                                                                                                           32% ███▎      
 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_object_not_empty[inputs0] ✓                                                                                                                     35% ███▌      
 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_array_type[inputs0] ✓                                                                                                                           38% ███▉      
 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_forbidden_complex_types[inputs0] ✓                                                                                                              41% ████▎     
 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_date_pattern[inputs0] ✓                                                                                                                         44% ████▌     
 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_date_format[inputs0] ✓                                                                                                                          47% ████▊     {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_defined_refs_exist_in_json_spec_file[inputs0] ✓                                                                                                 50% █████     {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_oauth_flow_parameters[inputs0] ✓                                                                                                                53% █████▍    Pulling docker image airbyte/source-pendo:latest
{"type": "LOG", "log": {"level": "WARN", "message": "\n We did not find the airbyte/source-pendo:latest image for this connector. This probably means this version has not yet been published to an accessible docker registry like DockerHub."}}
{"type": "LOG", "log": {"level": "WARN", "message": "\n We could not retrieve the previous connector spec as a connector runner for the previous connector version could not be instantiated."}}
{"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_backward_compatibility[inputs0] s                                                                                                               56% █████▋    {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestSpec.test_additional_properties_is_true[inputs0] ✓                                                                                                        59% █████▉    {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestConnection.test_check[inputs0] ✓                                                                                                                          62% ██████▎   {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/integration_tests/invalid_config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestConnection.test_check[inputs1] ✓                                                                                                                          65% ██████▌   {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestDiscovery.test_discover[inputs0] ✓                                                                                                                        68% ██████▊   {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestDiscovery.test_defined_cursors_exist_in_schema[inputs0] ✓                                                                                                 71% ███████▏  {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestDiscovery.test_defined_refs_exist_in_schema[inputs0] ✓                                                                                                    74% ███████▍  {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestDiscovery.test_defined_keyword_exist_in_schema[inputs0-allOf] ✓                                                                                           76% ███████▋  {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestDiscovery.test_defined_keyword_exist_in_schema[inputs0-not] ✓                                                                                             79% ███████▉  {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestDiscovery.test_primary_keys_exist_in_schema[inputs0] ✓                                                                                                    82% ████████▎ {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestDiscovery.test_streams_has_sync_modes[inputs0] ✓                                                                                                          85% ████████▌ {"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestDiscovery.test_additional_properties_is_true[inputs0] ✓                                                                                                   88% ████████▉ Pulling docker image airbyte/source-pendo:latest
{"type": "LOG", "log": {"level": "WARN", "message": "\n We did not find the airbyte/source-pendo:latest image for this connector. This probably means this version has not yet been published to an accessible docker registry like DockerHub."}}
{"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}
{"type": "LOG", "log": {"level": "WARN", "message": "\n We could not retrieve the previous discovered catalog as a connector runner for the previous connector version (airbyte/source-pendo:latest) could not be instantiated."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestDiscovery.test_backward_compatibility[inputs0] s                                                                                                          91% █████████▎{"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}
{"type": "LOG", "log": {"level": "INFO", "message": "The configured catalog is built from a custom configured catalog."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestBasicRead.test_read[inputs0] ✓                                                                                                                            94% █████████▌{"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}
{"type": "LOG", "log": {"level": "ERROR", "message": "Docker container failed, code 1, error:\n{\"type\": \"TRACE\", \"trace\": {\"type\": \"ERROR\", \"emitted_at\": 1678857957554.178, \"error\": {\"message\": \"Something went wrong in the connector. See the logs for more details.\", \"internal_message\": \"2 validation errors for ConfiguredAirbyteCatalog\\nstreams -> 0 -> sync_mode\\n  value is not a valid enumeration member; permitted: 'full_refresh', 'incremental' (type=type_error.enum; enum_values=[<SyncMode.full_refresh: 'full_refresh'>, <SyncMode.incremental: 'incremental'>])\\nstreams -> 0 -> destination_sync_mode\\n  value is not a valid enumeration member; permitted: 'append', 'overwrite', 'append_dedup' (type=type_error.enum; enum_values=[<DestinationSyncMode.append: 'append'>, <DestinationSyncMode.overwrite: 'overwrite'>, <DestinationSyncMode.append_dedup: 'append_dedup'>])\", \"stack_trace\": \"Traceback (most recent call last):\\n  File \\\"/airbyte/integration_code/main.py\\\", line 13, in <module>\\n    launch(source, sys.argv[1:])\\n  File \\\"/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py\\\", line 131, in launch\\n    for message in source_entrypoint.run(parsed_args):\\n  File \\\"/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py\\\", line 119, in run\\n    config_catalog = self.source.read_catalog(parsed_args.catalog)\\n  File \\\"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/source.py\\\", line 89, in read_catalog\\n    return ConfiguredAirbyteCatalog.parse_obj(self._read_json_file(catalog_path))\\n  File \\\"/usr/local/lib/python3.9/site-packages/pydantic/main.py\\\", line 521, in parse_obj\\n    return cls(**obj)\\n  File \\\"/usr/local/lib/python3.9/site-packages/pydantic/main.py\\\", line 341, in __init__\\n    raise validation_error\\npydantic.error_wrappers.ValidationError: 2 validation errors for ConfiguredAirbyteCatalog\\nstreams -> 0 -> sync_mode\\n  value is not a valid enumeration member; permitted: 'full_refresh', 'incremental' (type=type_error.enum; enum_values=[<SyncMode.full_refresh: 'full_refresh'>, <SyncMode.incremental: 'incremental'>])\\nstreams -> 0 -> destination_sync_mode\\n  value is not a valid enumeration member; permitted: 'append', 'overwrite', 'append_dedup' (type=type_error.enum; enum_values=[<DestinationSyncMode.append: 'append'>, <DestinationSyncMode.overwrite: 'overwrite'>, <DestinationSyncMode.append_dedup: 'append_dedup'>])\\n\", \"failure_type\": \"system_error\"}}}\n"}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py::TestBasicRead.test_airbyte_trace_message_on_failure[inputs0] ✓                                                                                                97% █████████▊{"type": "LOG", "log": {"level": "INFO", "message": "Using /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/secrets/config.json as configuration. It is the most recent version."}}
{"type": "LOG", "log": {"level": "INFO", "message": "The configured catalog is built from a custom configured catalog."}}

 airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_full_refresh.py::TestFullRefresh.test_sequential_reads[inputs0] ✓                                                                                                     100% ██████████
{"type": "LOG", "log": {"level": "INFO", "message": "/Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo - Connector Acceptance Test run - 64d9c7dca2a157e1144149b9b7100f34e0c81443 - PASSED"}}

=============================================================================================================================== warnings summary ===============================================================================================================================
airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py: 26 warnings
airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_full_refresh.py: 1 warning
  /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/.venv/lib/python3.9/site-packages/docker/utils/utils.py:52: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    s1 = StrictVersion(v1)

airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py: 26 warnings
airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_full_refresh.py: 1 warning
  /Users/bartdev/Playground/airbyte/airbyte-integrations/connectors/source-pendo/.venv/lib/python3.9/site-packages/docker/utils/utils.py:53: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    s2 = StrictVersion(v2)

-- Docs: https://docs.pytest.org/en/stable/warnings.html
=========================================================================================================================== short test summary info ============================================================================================================================
SKIPPED [1] ../../bases/connector-acceptance-test/connector_acceptance_test/plugin.py:63: Skipping TestIncremental.test_two_sequential_reads: This connector does not implement incremental sync
SKIPPED [1] ../../bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py:103: The previous connector image could not be retrieved.
SKIPPED [1] ../../bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py:512: The previous connector image could not be retrieved.

Results (30.43s):
      32 passed
       2 skipped
```

File: airbyte-integrations/connectors/source-pendo/source_pendo/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,3133 @@
...
+                      type: string
+                    001ucoitoqlbx:
+                      type: string
+                    002rjbvc6osjr:
+                      type: string
+                    002s9w5ylxoyj:
+                      type: string
+                    002xuuhdcdo9l:
+                      type: string
+                    003k9r8idkmqq:
```
Comment (sajarin) [inline airbyte-integrations/connectors/source-pendo/source_pendo/manifest.yaml:1234]: What do these random string ids refer to?

File: airbyte-integrations/connectors/source-pendo/source_pendo/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,3133 @@
...
+                            poll_czx5ep1jrtu_14nCfODWTVf1OGZXupmQTzBdbHU_pollResponseAllTime:
+                              type: string
+                            poll_82cfeahyjdj_d0fWyIuRnhHZn2YFVltKospMBbU_pollResponseWithinDateRange:
+                              type: string
+                            poll_czx5ep1jrtu_14nCfODWTVf1OGZXupmQTzBdbHU_pollResponseWithinDateRange:
+                              type: string
+                            poll_jq84am6dxpm_14nCfODWTVf1OGZXupmQTzBdbHU_pollResponseWithinDateRange:
+                              type: string
+                            poll_x97q4ruspri_d0fWyIuRnhHZn2YFVltKospMBbU_pollResponseWithinDateRange:
+                              type: string
```
Comment (sajarin) [inline airbyte-integrations/connectors/source-pendo/source_pendo/manifest.yaml:753]: What do these values refer to? They seem random but they also appear elsewhere here so just trying to get an idea.

**Review by sajarin — CHANGES_REQUESTED** [review]
Some comments: 

Also please resolve the merge conflict in the source_definitions.yaml file

**Comment by misteryeo** [comment]
**Issue was linked to Harvestr Discovery:**  [New Source: Pendo](https://app.harvestr.io/components/0/list/hQPNTlqZm)

**Review by marcosmarxm — APPROVED** [review]

**Review by sajarin — APPROVED** [review]

**Comment by marcosmarxm** [comment]
Published in https://github.com/airbytehq/airbyte/pull/24714

**Pull request closed without merging**